tutorials/index.rst

```diff
@@ -0,0 +1,78 @@
+.. Kornia Tutorials documentation master file, created by
+   sphinx-quickstart on Thu Jan 21 19:52:01 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Kornia Tutorials's documentation!
+============================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: TUTORIALS
+
+   🏠 Main page 🏠 <https://kornia.readthedocs.io/en/latest>
+
+.. toctree::
+   :maxdepth: 2
+   :caption: BASIC
+   
+   _nbs/hello_world_tutorial
```

Comment:
I don't know if there is a way to do this without the explicit folder (_nbs/) other than to root all laptops

Reply:
not that i'm aware of right now